kube/stateful_set.go

```diff
@@ -86,12 +86,13 @@ func getVolumeClaims(role *model.InstanceGroup, createHelmChart bool) []helm.Nod
 		}
 
 		meta := helm.NewMapping("name", volume.Tag)
-		annotationList := helm.NewMapping()
-		annotationList.Add(VolumeStorageClassAnnotation, storageClass)
-		for key, value := range volume.Annotations {
-			annotationList.Add(key, value)
+		if len(volume.Annotations) > 0 {
```

Comment:
Aaron always hated it, but the `helm` code uses reflection to dynamically deal with data types, so you can simplify this as:

```
		if len(volume.Annotations) > 0 {
			meta.Add("annotations", helm.NewNode(volume.Annotations))
		}
```

Reply:
Ok.

Reply:
Done with commit 80d42a64006c290843a576942e286a7911df71e4